interface/patient_file/summary/demographics.php

```diff
@@ -6,11 +6,9 @@
  * @package OpenEMR
  * @link    http://www.open-emr.org
  * @author  Brady Miller <brady.g.miller@gmail.com>
- * @author  Sharon Cohen <sharonco@matrix.co.il>
- * @author  Stephen Waite <stephen.waite@cmsvt.com>
```

Comment:
This likely means that recently added code was missed

Reply:
Apart from the authors the rest of the script should be the same. In order to have a common 'header-title ' that shows the eRx buttons, patient portal buttons and deceased notification across most of the pages I split that code into the `interface/patient_file/summary/dashboard_header.php` so that it could be included on all the pages that need it via

           <?php
            if ($thisauth) {
                $header_title = xl('Medical Record Dashboard of');?>
                <div class="row">
                    <div class="col-sm-12">
                        <?php
                            $portal_login_href = "create_portallogin.php";
                            require_once("dashboard_header.php");
                        ?>
                    </div>
                </div>
            <?php
            } // $thisauth
            ?>

The navbar  is included with 
`require_once("dashboard_nav.php");`

In order for the **NewCrop Account Status**  and **Create Onsite Portal Credentials** buttons to work across all pages two jquery functions from demographics php :
`$(".iframe1").on('click', function(e)`
`$(".small_modal").on('click', function(e) `
were placed in 
`interface/patient_file/erx_patient_portal_js.php`

The deceased notification message has been re-written

Hopefully this will help you in your review :)

 I will correct the omission  of the author notification